app/styles/modules/_custom-rows.scss

```diff
@@ -171,15 +171,29 @@ ul.links {
 }
 
 #permissions {
-  .input-row {
-    text-align: left;
-    margin: 30px 0;
+  fieldset {
+    border: none;
+  }
+
+  .avatar-wrapper {
+    height: 50px;
+    width: 50px;
   }
 
-  input[disabled] {
-    background-color: $input-background-disabled-color;
-    border-color: $input-border-disabled-color;
-    color: $input-disabled-color;
+  .fxa-checkbox {
+    color: #8a9ba8;
```

Comment:
I was unsure whether these changes should be universal and go into the fxa-checkbox repo.